presto-main/src/test/java/com/facebook/presto/failureDetector/TestHeartbeatFailureDetector.java

```diff
@@ -88,6 +93,34 @@ public void configure(Binder binder)
         assertTrue(detector.getFailed().isEmpty());
     }
 
+    @Test
```

Comment:
@satybald Can you update the tests to merge them into the single test below? This one tests both success and failure, and do a round trip to test ser/deser.

```
    @Test
    public void testHeartbeatStatsSerialization()
            throws Exception
    {
        ObjectMapper objectMapper = new ObjectMapperProvider().get();
        Stats stats = new Stats(new URI("http://example.com"));
        String serialized = objectMapper.writeValueAsString(stats);
        JsonNode deserialized = objectMapper.readTree(serialized);
        assertTrue(deserialized.get("lastFailureInfo").isNull());

        stats.recordFailure(new SocketTimeoutException("timeout"));
        serialized = objectMapper.writeValueAsString(stats);
        deserialized = objectMapper.readTree(serialized);
        assertFalse(deserialized.get("lastFailureInfo").isNull());
        assertEquals(deserialized.get("lastFailureInfo").get("type").asText(), SocketTimeoutException.class.getName());
    }
```